src/model/cards/legends/cirrus_windfall.ts

```diff
@@ -0,0 +1,35 @@
+import { Card } from "../../card";
+import { GameState } from "../../gameState";
+import { toGameText } from "../../gameText";
+import {
+  CardName,
+  CardType,
+  ExpansionType,
+  GameInput,
+  ResourceType,
+} from "../../types";
+
+export const cirrus_windfall: ConstructorParameters<typeof Card>[0] = {
+  expansion: ExpansionType.LEGENDS,
+  name: CardName.CIRRUS_WINDFALL,
+  upgradeableCard: CardName.POSTAL_PIGEON,
+  cardType: CardType.TRAVELER,
+  cardDescription: toGameText(
+    "You may play 1 CARD worth up to 3 POINT for free."
```

Comment:
To get the yellow circle w/ the number, you can use `{ type: "points", value: 3 }` 😄

Reply:
yeah, i did a rush job on the card descriptions first, will be cleaning them up